charts/prometheus-adapter/templates/custom-metrics-cluster-role-binding-hpa.yaml

```diff
@@ -6,11 +6,13 @@ as otherwise this binding will point to not existing role.
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
+  {{- if .Values.customAnnotations }}
+  annotations:
+  {{- toYaml .Values.customAnnotations | nindent 4 }}
+  {{- end }}
   labels:
     app: {{ template "k8s-prometheus-adapter.name" . }}
```

Comment:
this is redundant with: "app.kubernetes.io/part-of"

Reply:
done